index.html

```diff
@@ -0,0 +1,341 @@
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+<footer>
+  <div class="contact">
+    <div class="row">
+
+      <!--Grid column-->
+      <div class="col-md-6 mb-4 p-6">
+        <div class="form-text d-flex justify-content-center">
+        <h4>I'm always interested in hearing about <br>new projects, so if you'd like to chat<br> please get in touch</h4></div>
+
+    
+    </div>
+        <!--Grid column-->
+        <div class="col-md-6 mb-4">
+        <form action="https://formspree.io/f/xvoddqjb" method="POST">
+          <div class="form-group">
+            <label for="exampleFormControlInput1">Name</label>
+            <input type="text" name="name" required class="form-control" id="exampleFormControlInput1" placeholder="John Smith" maxlength="30">
+          </div>
+          <DIV class="form-group">
+            <label for="exampleFormControlInput1">Email </label>
+            <input type="email" name="_replyto" required class="form-control" id="exampleFormControlInput2" placeholder="name@example.com">
+          </DIV>
+
+          <div class="form-group">
+            <label for="exampleFormControlTextarea1">Message</label>
+            <textarea name="message" class="form-control" id="exampleFormControlTextarea1" rows="3" maxlength="500" required></textarea>
```

Comment:
- [ ] Kindly add a `placeholder` attribute to all fields in your form, currently your `<textarea>` does not have one 🤓
  - A `placeholder` is recommended because it describes the expected value of an input field/text area.